### PR TITLE
Create metadata for OVC program

### PR DESCRIPTION
**Pull request by brandones**
This creates a stub for the OVC program, as well as the encounter types for the intake and follow-up forms.

Relates to
[UHM-4979 OVC Intake form](https://pihemr.atlassian.net/browse/UHM-4979)
[UHM-4986 OVC Program config](https://pihemr.atlassian.net/browse/UHM-4986)

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/metadata/core/EncounterTypes.java [hunk 1]
```diff
@@ -319,6 +319,18 @@
 		public String description() { return "COVID-19 Discharge note at health facility"; }
 	};
 
+	public static EncounterTypeDescriptor OVC_INTAKE  = new EncounterTypeDescriptor() {
+		public String uuid() { return "651d4359-4463-4e52-8fde-e62876f90792"; }
+		public String name() { return "OVC Intake"; }
+		public String description() { return "OVC intake form"; }
```
Comment (lnball) [inline api/src/main/java/org/openmrs/module/pihcore/metadata/core/EncounterTypes.java:325]: I would change the description to say "Orphan Vulnerable Child (OVC)"

File: api/src/main/java/org/openmrs/module/pihcore/metadata/core/EncounterTypes.java [hunk 2]
```diff
@@ -319,6 +319,18 @@
...
+	};
+
+	public static EncounterTypeDescriptor OVC_FOLLOWUP  = new EncounterTypeDescriptor() {
+		public String uuid() { return "f8d426fd-132a-4032-93da-1213c30e2b74"; }
+		public String name() { return "OVC Follow-up"; }
+		public String description() { return "OVC follow-up form"; }
```
Comment (lnball) [inline api/src/main/java/org/openmrs/module/pihcore/metadata/core/EncounterTypes.java:331]: same as above

**Review by lnball — CHANGES_REQUESTED** [review]
Tiny change to form descriptions.  Apologies @brandones

**Comment by brandones** [comment]
Thanks @lnball !

**Pull request merged**